src/components/ConfigContext.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Icon, icons } from './Icon';
+
+export interface ConfigValue {
+  icons: typeof icons;
+  Icon: typeof Icon;
+}
+
+export const ConfigContext = React.createContext<ConfigValue>({ Icon, icons });
```

Comment:
We should probably break this context down into smaller parts once we add hooks.